README.md

```diff
@@ -1,11 +1,41 @@
 # Arigato 
+Control Flow Examples
```

Comment:
This line will throw an error because the runtime will not be able to interpret it. In the future, you can use comments like this:

```
// Control Flow Examples
```